deploy/crds/app.kiegroup.org_v1alpha1_kogitodataindex_cr.yaml

```diff
@@ -9,16 +9,16 @@ spec:
   replicas: 1
   # image to use for this deploy
   image:
-    tag: "0.7.0"
+    tag: "0.8.0"
   # httpPort: 8080
   # details about the kafka connection
   kafka:
     # let's leave this burden to KogitoInfra CR to deploy a new Kafka instance for us
-    useKogitoInfra: true
+    useKogitoInfra: false
```

Comment:
why did you switch to false here ?
dataindex will need kafka and infinispan or ?

Reply:
Because of the CI on OperatorHub. Since they don't implement the dependency management there, the Data Index will fail to deploy and will fail the PR

Reply:
oki

Reply:
Although I'll work on this for the next version. The Operator will handle with this kind of failures and will have a nice status message.